helm/flowforge/README.md

```diff
@@ -67,6 +67,13 @@ Enables FlowForge Telemetry
  - `forge.telemetry.posthog.apikey` enables posthog logging if set (no default)
  - `forge.telemetry.posthog.capture_pageview` (default `true`)
 
+ ### Support
+
+ Enables HubSpot support widget in the FlowForge app
+
+ - `forge.support.enabled` enables support widget (default `false`)
+ - `forge.support.hubspot` HubSpot tracking code
```

Comment:
should this be `forge.support.hubspot.trackingcode`?

Reply:
or does it not need to map to the `yml` structure, and given that there is only one nested value atm in the `hubspot` section, makes sense to squash

Reply:
yes, I compressed this, the values are mapped to the right place in the `configmap.yml` template